product-catalog/app/productcatalog/pages/ProductDetailPageContent.java

```diff
@@ -0,0 +1,54 @@
+package productcatalog.pages;
+
+import common.pages.*;
+
+import java.util.List;
+
+public class ProductDetailPageContent extends PageContent {
+
+    private final String additionalTitle;
+    private final PdpStaticData staticData;
+    private final List<LinkData> breadcrumbs;
+    private final List<ImageData> gallery;
+    private final ProductData productData;
+    private final List<ShippingRateData> deliveryData;
+    private final List<ProductThumbnailData> suggestions;
+
+    public ProductDetailPageContent(final String additionalTitle, final PdpStaticData staticData, final List<LinkData> breadcrumbs, final List<ImageData> gallery, final ProductData productData, final List<ShippingRateData> deliveryData, final List<ProductThumbnailData> suggestions) {
+        this.additionalTitle = additionalTitle;
+        this.staticData = staticData;
+        this.breadcrumbs = breadcrumbs;
+        this.gallery = gallery;
+        this.productData = productData;
+        this.deliveryData = deliveryData;
+        this.suggestions = suggestions;
+    }
+
+    public PdpStaticData getStatic() {
+        return staticData;
+    }
+
+    public String additionalTitle() {
+        return additionalTitle;
+    }
+
+    public List<LinkData> getBreadcrumb() {
```

Comment:
`breadcrumb` is the expected name defined by the JSON data:
https://github.com/sphereio/sphere-sunrise-design/blob/master/input/templates/pdp.json#L868

The current value resolver looks for methods with a prefix `getBreadcrumb` or `isBreadcrumb`. So it needs to be called like that. We can change it in the JSON data, but there it fits better simply `breadcrumb`.